app/src/main/java/com/example/marvelapp/presentation/characters/CharactersViewHolder.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.marvelapp.presentation.characters
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import br.com.msmlabs.core.domain.model.Character
+import com.bumptech.glide.Glide
+import com.example.marvelapp.R
+import com.example.marvelapp.databinding.ItemCharacterBinding
+
+class CharactersViewHolder(
+    itemCharacterBinding: ItemCharacterBinding
+) : RecyclerView.ViewHolder(itemCharacterBinding.root) {
+
+    private val textName = itemCharacterBinding.tvName
+    private val imageCharacter = itemCharacterBinding.ivCharacter
+
+    fun bind(character: Character) {
+        textName.text = character.name
+        Glide.with(itemView)
```

Comment:
Implement abstraction of image loading library in case we want to change it in the future